### PR TITLE
Ref #3177: Add more regions for Ecosia.

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

Related desktop ticket https://github.com/brave/brave-browser/issues/13511

This pull request references #3177 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by kylehickinson — APPROVED** [review]

**Review by Brandon-T — APPROVED** [review]

**Review by soner-yuksel — APPROVED** [review]

File: Client/Frontend/Browser/Search/InitialSearchEngines.swift [hunk 1]
```diff
@@ -56,7 +56,8 @@ class InitialSearchEngines {
     static let ddgDefaultRegions = ["DE", "AU", "NZ", "IE"]
     static let qwantDefaultRegions = ["FR"]
     static let yandexDefaultRegions = ["AM", "AZ", "BY", "KG", "KZ", "MD", "RU", "TJ", "TM", "TZ"]
-    static let ecosiaDefaultRegions = ["US", "GB", "FR", "DE", "NL", "BE", "CH", "SE"]
+    static let ecosiaDefaultRegions = ["AT", "AU", "BE", "BY", "CA", "DK", "ES", "FI", "GR", "HU", "IT",
```
Comment (bsclifton) [inline Client/Frontend/Browser/Search/InitialSearchEngines.swift]: - I don't think `BY` is supposed to be in here; maybe copy/paste error?
- others look good!

**Review by bsclifton — CHANGES_REQUESTED** [review]
comment left; I think one was put by accident, otherwise LGTM! 😄

**Comment by iccub** [comment]
@bsclifton thanks, removed 'BY', added it by mistake

**Review by bsclifton — APPROVED** [review]
Perfect 😄👌

**Pull request merged**